proxyvisor/proxyvisor.go

```diff
@@ -0,0 +1,21 @@
+package proxyvisor
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Provision() (string, error) {
+	// Simulate proxyvisor whilst we wait for it to be released by returning a random 62 char string
+	return random(62), nil
+}
+
+func random(strlen int) string {
```

Comment:
its literally there to replace the proxyvisor whilst its being worked on. This will be removed later on